### DISCUSSION TITLE
Great Up date

### DISCUSSION THREAD

**Discussion by Cordia709** [YimMenu/YimMenu#1413] [General]
Works fine apart from loading screen ,gives a little Error in case you are not aware off it.I tried to fix it but out of my range ,as i am still learning coding.
the error is ([FATAL/tunables_service.cpp:87] Failed to cache tunables) I don't think it has the .hpp file ,but i didn't want to mess with it.
Cheers
 John
Cordia790.

**Comment by aoqia194** [comment]
It seems that YimMenu works fine regardless of if it fails to cache the tunables or not. Personally, since the latest update, YimMenu fails to cache tunables like you have shown here. It shouldn't affect gameplay or performance of YimMenu from my understanding.

**Reply by Yimura** [reply]
Also weird to frustrate about an error while it doesn't really affect the experience of the menu.
